packages/suite/src/support/messages.ts

```diff
@@ -6678,4 +6678,9 @@ export default defineMessages({
         id: 'FAILED_TO_DISABLE_TOR',
         defaultMessage: 'Failed to disable Tor',
     },
+    TR_FIRMWARE_HASH_MISMATCH: {
+        id: 'TR_FIRMWARE_HASH_MISMATCH',
+        defaultMessage:
+            'Do not use this deivce! This device might have been hacked. Conntact Trezor support immediately.',
```

Comment:
typos: 
```suggestion
            'Do not use this device! This device might have been hacked. Contact Trezor support immediately.',
```

Reply:
Forget the typos, the approved message is:

"Your Trezor is running unofficial firmware. Please contact help@trezor.io immediately."
```suggestion
            'Your Trezor is running unofficial firmware. Please contact help@trezor.io immediately.',
```

Reply:
fixed and squashed directly